pkg/storaged/pages.jsx

```diff
@@ -591,7 +591,7 @@ export const PageTable = ({ emptyCaption, aria_label, pages, crossrefs, sorted,
                     <CardBody>
                         <Split hasGutter>
                             { icon && <SplitItem>{icon}</SplitItem> }
-                            <SplitItem isFilled><strong>{name}</strong>{info}</SplitItem>
+                            <SplitItem isFilled><strong className="pf-v5-c-truncate"><span className="pf-v5-c-truncate__start">{name}</span></strong>{info}</SplitItem>
```

Comment:
This doesn't put the ellipsis into the middle anymore, which I thought was very nice. The HTML version should also support that, so let's bring that back.

This will probably be enough code to warrant making a new custom React element.

Reply:
I'm not a big fan of duplicating PatternFly elements in our own code for one use case. I don't think Patternfly will ever accept a `Truncate` adding an option without a tooltip as that's an anti-pattern (rightfully so).

Reply:
> for one use case

But we use it twice, no?

> I don't think Patternfly will ever accept a Truncate adding an option without a tooltip as that's an anti-pattern (rightfully so).

IMO it's a bug that a tooltip is always shown, even when there is no truncation. This makes `Truncate` unusable, unfortunately, even if it would be the right thing.

Reply:
I have filed an issue: https://github.com/patternfly/patternfly/issues/6253